external/harvestPackages/harvestPackages.depproj

```diff
@@ -4,143 +4,191 @@
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.targets))\dir.targets" />
 
   <PropertyGroup>
-    <TargetFrameworks>netstandard1.6</TargetFrameworks>
+    <TargetFrameworks>netstandard2.0</TargetFrameworks>
   </PropertyGroup>
 
   <ItemGroup>
     <PackageReference Include="Microsoft.CSharp">
-      <Version>4.3.0</Version>
+      <Version>4.4.0</Version>
     </PackageReference>
     <PackageReference Include="Microsoft.VisualBasic">
-      <Version>10.1.0</Version>
+      <Version>10.2.0</Version>
     </PackageReference>
     <PackageReference Include="Microsoft.Win32.Registry">
-      <Version>4.3.0</Version>
+      <Version>4.4.0</Version>
     </PackageReference>
     <PackageReference Include="Microsoft.Win32.Registry.AccessControl">
-     <Version>4.3.0</Version>
+      <Version>4.4.0</Version>
     </PackageReference>
     <PackageReference Include="System.Buffers">
-      <Version>4.3.0</Version>
+      <Version>4.4.0</Version>
+    </PackageReference>
+    <PackageReference Include="System.CodeDom">
```

Comment:
We must harvest every package we've ever shipped before, even if we aren't redistributing assets we calculate the old supported frameworks to make sure the new supported frameworks are compatible.